src/components/LoginForm/LoginForm.js

```diff
@@ -18,25 +23,44 @@ export class LoginForm extends Component {
   }
 
   handleLogin = (e) => {
-    const { addUser } = this.props;
     const { username, password } = this.state;
     const user = {
       username,
       password
     };
-
+    this.setState({ error: '', loading: true });
+    
     e.preventDefault();
+    this.signIn(user);
+  }
 
-    addUser(user);
+  signIn = async (user) => {
```

Comment:
Awesome work! will this make a live query with our backend?

Reply:
yes!